### ISSUE TITLE
--gtid should include relay-log-recovery

### ISSUE THREAD

**Issue by datacharmer** [datacharmer/dbdeployer#35]
The option `--gtid` wants to put the server in a crash-safe replication state. For that reason, it includes:

```
master-info-repository=table
relay-log-info-repository=table
gtid_mode=ON
log-slave-updates
enforce-gtid-consistency
```
As shown in [MySQL bug#92093](https://bugs.mysql.com/bug.php?id=92093), crash-safe state should include the option `relay-log-recovery=ON`

**Comment by jfg956** [comment]
As a small comment, "master-info-repository=table" is not needed for replication crash-safety.  I do not think it should be removed, just pointing-out it is not needed.  More about this in [1].

[1]: https://jfg-mysql.blogspot.com/2016/08/discussion-about-sync-master-info-and-replication-parameters.html

**Comment by datacharmer** [comment]
Fixed in release 1.11.0.
Thanks to @jfg956 for reporting the problem
